recipes/avahi/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+project(test_package LANGUAGES C)
 
 find_package(Avahi CONFIG REQUIRED)
  
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} Avahi::compat-libdns_sd)
+target_link_libraries(${PROJECT_NAME} PRIVATE avahi::compat-libdns_sd)
```

Comment:
Was it previously wrong?

Reply:
Yes, I believe it was by changing the default from `Avahi` to `avahi` for the old CMake generators. Avahi does not provide any sort of CMake Config nor does CMake provide an official Find Module. In this case, the defaults for the `CMakeDeps` generator should be used, right?

Reply:
@jwillikers Correct!